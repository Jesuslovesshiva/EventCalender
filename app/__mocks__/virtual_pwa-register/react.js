import {useState} from 'react'

const useRegisterSW = jest.fn(() => {
  const [needRefresh, setNeedRefresh] = useState(true)

  return {
    needRefresh: [needRefresh, setNeedRefresh],
    updateServiceWorker: () => {}
  }
})

// eslint-disable-next-line import/prefer-default-export
export {useRegisterSW}
