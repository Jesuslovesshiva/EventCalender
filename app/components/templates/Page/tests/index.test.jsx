import {render} from '@testing-library/react'

import PageTemplate from '../index'

const renderComponent = () =>
  render(
    <PageTemplate header="Header" footer="Footer">
      Children
    </PageTemplate>
  )

describe('PageTemplate', () => {
  it('should match the snapshot', () => {
    const {container} = renderComponent()
    expect(container.firstChild).toMatchSnapshot()
  })
})
