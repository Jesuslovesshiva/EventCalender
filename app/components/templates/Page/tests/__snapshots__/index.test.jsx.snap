// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTemplate should match the snapshot 1`] = `
<div
  class="flex flex-col h-full bg-background overflow-x-auto"
>
  Header
  <div
    class="flex-1 overflow-y-auto"
  >
    Children
  </div>
  Footer
</div>
`;
